### PR TITLE
Unblacklist a few perl packages

### PR DESCRIPTION
**Pull request by abretaud**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/perl-xml-libxml/meta.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+  version: "2.0132"
 
 source:
-  url: https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXML-2.0124.tar.gz
-  md5: 30436b85454fba3ae5f6494df598e65c
+  url: https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXML-2.0132.tar.gz
+  md5: 43546fd9a3974f19323f9fb04861ece9
 
 build:
   number: 1
```
Comment (bgruening) [inline recipes/perl-xml-libxml/meta.yaml]: can we reset this here to 0 - you can use [ci skip] if you do it - it should not influence the previous test

**Comment by abretaud** [comment]
@bioconda/core oops, I forgot to remove "ci skip" from the squash comment, the bioconda-upload-master jobs were skipped. Do you have a way to run them anyway or should I do something?

**Comment by bgruening** [comment]
If we are lucky the nightly builds are catching those and rebuild them.

**Comment by bgruening** [comment]
I also merged master into bulk, this should also help. Lets see if tm morning everything is green and sunny.

Thanks @abretaud!

**Comment by abretaud** [comment]
Woohoo, it worked! thanks!
green and sunny

**Pull request merged**